embulk-input-postgresql/src/main/java/org/embulk/input/postgresql/PostgreSQLInputConnection.java

```diff
@@ -76,4 +78,19 @@ public void close() throws SQLException
             // TODO close?
         }
     }
+
+    private void setStatementTimeout(int statementTimeout)
+        throws SQLException
+    {
+        if (statementTimeout > -1) {
+            Statement stmt = connection.createStatement();
+            try {
+                String sql = "SET statement_timeout TO " + quoteIdentifierString(String.valueOf(statementTimeout));
```

Comment:
Looking at other SQL methods (e.g. `newBatchSelect` above), it could be nice to have a log like:

```
logger.info("SQL: " + sql);
```